website-frontend/src/components/NavBar.js

```diff
@@ -1,22 +1,35 @@
 // @nhomnhomm Complete this
+import {Link} from 'react-router-dom'
+import logo from '../images/logo.png'
 
-import { Link } from "react-router-dom"
-
-const NavBtn = ({path, text}) => {
+const NavLink = ({path, text}) => {
   return (
-    <Link to={ path }>
-      <button>
-        { text }
-      </button>
+    <Link to = {path} className='nav-item nav-link pb-0 pt-0'>
```

Comment:
Why do we need `pb-0` and `pt-0` em? I tried removing that, and it looks quite good.

Reply:
You may want to read about `px-?` and `py-?` as well.

Reply:
I think it was because the padding was too big (which it is not anymore), but honestly I did not remember why:) so once again thank you anh.